README.md

```diff
@@ -39,7 +39,9 @@ If not running OS X, or you don't like Homebrew, you can use [pip](https://pip.p
 
 ```bash
 # Install Mackup with PIP
-pip install mackup
+pip install --upgrade mackup
+
+Note: If a previous version of Mackup already exists, it will be upgraded to the latest version
```

Comment:
It's not a command, it should be a comment. Also the comment should be before the commands it applies to. You don't want to know what a command does after you execute it.